xsuite/src/data/index.test.ts

```diff
@@ -159,6 +159,8 @@ beforeAll(async () => {
     gasLimit: 10_000_000,
   });
   complexContractState = await contract.getSerializableAccount();
+  complexContractState.codeHash =
```

Comment:
Is there a reason not having used the codeHash that was suggested? The suggested codeHash has the advantage to make it very explicit that the codeHash is made up. With the current one, someone's might wonder why this codeHash is used.